packages/devextreme/js/ui/drop_down_editor/ui.drop_down_list.d.ts

```diff
@@ -45,7 +45,10 @@ export interface SelectionChangedInfo<T = any> {
  * @docid
  * @hidden
  */
-export interface dxDropDownListOptions<TComponent> extends DataExpressionMixinOptions<TComponent>, dxDropDownEditorOptions<TComponent> {
+export interface dxDropDownListOptions<TComponent> extends DataExpressionMixinOptions<TComponent>, Omit<
```

Comment:
Why? We need this API in SelectBox and TagBox inherited from ddList.
Look at generation in frameworks - there is no new API for these components now - [link](https://github.com/DevExpress/DevExtreme/pull/30723/files#diff-4c797649be6ce2ad6ad606625607620f63965ca71c01c688f46fa54390cfe950R203)

Reply:
> Why? We need this API in SelectBox and TagBox inherited from ddList.
We don't need this option in all other ddLists. Therefore, it was added separately for SelectBox: e3f3a6143349e9ee56cca0b9c07ad7f779000e68

Reply:
Discussed in voice - we think it's better to have types specifed on the same level where implementation is